python/src/cerebellum/browser.py

```diff
@@ -9,7 +9,7 @@
 from abc import ABC, abstractmethod
 from dataclasses import dataclass
 from enum import StrEnum
-from typing import Any, Literal
+from typing import Any, Dict, List, Literal
```

Comment:
@theredsix for maintained a typed library, I was pushing for the built-ins that can be leveraged for types, particularly items like `Dict` -> `dict` and `List` -> `list`.

Reply:
I used the older types for backward compatibility with python 3.8. We can switch to lowercase if we bump up the minimum supported language version.

Reply:
I would recommend we bump minimum version to `3.9` as `3.8` officially reached EOL in October this year.

Reply:
Agreed, set to 3.9